### ISSUE TITLE
bitcoin-symbol.svg has two issues with the svg path

### ISSUE THREAD

**Issue by lucvanloon** [BitcoinDesign/Guide#1128]
**Describe the bug**
The [bitcoin-symbol.svg](https://bitcoin.design/assets/images/guide/getting-started/visual-language/bitcoin-symbol.svg) has two issues in the svg. See images below:

![image](https://github.com/user-attachments/assets/454bac50-9b76-4260-a042-ebef618c3f90)
![image](https://github.com/user-attachments/assets/f5571467-5d2f-4916-8b00-91100a60cba5)

This can be fixed with the following svg:

`<svg width="144" height="144" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path d="M144 72c0-39.764-32.235-72-72-72C32.236 0 0 32.236 0 72c0 39.765 32.236 72 72 72 39.765 0 72-32.235 72-72Z" fill="#F7931A"/><path d="m49.738 34.218-2.101 8.442s5.863 1.341 5.742 1.431c3.2.797 3.78 2.92 3.68 4.59l-8.864 35.55c-.387.968-1.386 2.43-3.618 1.877.08.112-5.738-1.427-5.738-1.427l-3.92 9.036c3.87.963 12.191 3.033 15.908 4.023l-3.267 13.131 7.889 1.967 3.24-12.987c2.16.576 4.248 1.116 6.3 1.629l-3.231 12.928 7.902 1.971 3.267-13.099c13.473 2.547 23.602 1.521 27.873-10.661 3.438-9.81-.171-15.471-7.263-19.161 5.161-1.188 9.049-4.59 10.093-11.601 1.427-9.576-5.863-14.728-15.84-18.166l3.24-12.983-7.902-1.966-3.15 12.635a293.112 293.112 0 0 0-6.331-1.485l3.172-12.721-7.888-1.971-3.24 12.978-15.93-3.947-.023-.013Zm35.83 52.942c-2.44 9.81-18.96 4.51-24.319 3.178l4.338-17.388c5.355 1.34 22.532 3.977 19.98 14.21ZM88.01 61.71c-2.232 8.918-15.975 4.391-20.43 3.284l3.928-15.772c4.46 1.107 18.82 3.186 16.502 12.488Z" fill="#fff"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h144v144H0z"/></clipPath></defs></svg>`

I think it would be nice if the bitcoindesign guide website has a good source of truth for the bitcoin logo.

**Comment by GBKS** [comment]
Yes, this is something we have discussed before. These minor issues were in the original SVG symbol that was [first proposed](https://web.archive.org/web/20130524191250/https://bitcointalk.org/index.php?topic=1631.0) in 2010. Some argue we should just keep them as a historic artifact of sorts, others don't care. What do you make of this?

**Comment by lucvanloon** [comment]
The artifacts provide no real purpose to the end user. It's just a risk that when displayed at large sizes the .svg doesn't look as good as it should. Imo this should be fixed.

**Comment by pranjal-coders** [comment]
Since these "artifacts" were part of the original SVG from 2010, keeping them maintains historical authenticity. <br>
I mean I understand the concern about displaying it at large sizes, but their presence doesn’t significantly impact usability or recognition. <br>
So, I think preserving them as a historical artifact is more reasonable.

**Comment by GBKS** [comment]
@oli28 proposed a fix that I just merged in.
